lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -129,14 +137,18 @@ def create_user(
         try:
             with self._connect_to_database() as connection, connection.cursor() as cursor:
                 cursor.execute(f"SELECT TRUE FROM pg_roles WHERE rolname='{user}';")
-                user_definition = f" WITH LOGIN{' SUPERUSER' if admin else ''} ENCRYPTED PASSWORD '{password}'"
+                user_definition = (
+                    f" WITH LOGIN{' SUPERUSER' if admin else ''} ENCRYPTED PASSWORD '{password}'"
+                )
                 if extra_user_roles:
                     user_definition += f' {extra_user_roles.replace(",", " ")}'
                 if cursor.fetchone() is not None:
                     statement = "ALTER ROLE {}"
                 else:
                     statement = "CREATE ROLE {}"
-                cursor.execute(sql.SQL(statement + user_definition + ";").format(sql.Identifier(user)))
+                cursor.execute(
+                    sql.SQL(statement + user_definition + ";").format(sql.Identifier(user))
```

Comment:
minor nit: can we use f-strings here?

Reply:
Yes, we can. Updated on https://github.com/canonical/postgresql-operator/pull/28/commits/790e369813b6d0b86e8fe418750f091bc2b7b911.